base/strings/basic.jl

```diff
@@ -443,25 +468,46 @@ end
 """
     nextind(str::AbstractString, i::Integer, n::Integer=1) -> Int
 
+* Case `n == 1`
+
 If `i` is in bounds in `s` return the index of the start of the character whose
-encoding starts after index `i`. If `i` is out of bounds in `s` return `i + 1`.
-If `n == 0` return `i`.
+encoding starts after index `i`. In other words, if `i` is the start of a
```

Comment:
I think this section need to be indented (and same below) if they should render as bullet points, right?

Reply:
fixed, thanks.